src/docs/docs/tools-and-technologies/nextjs.md

```diff
@@ -24,67 +49,75 @@ When working on a new component for the Telescope frontend, please consider the
 
 Here is the colour palette that we use to make our web site looks pretty and up-to-date.
 
+<Tabs className="unique-tabs">
+  <TabItem value="light" label="Light">
+
 ### Light Palette
 
-![#121D59](https://placehold.it/15/121D59/000000?text=+) **Primary**
+<Palette color="#121D59">Primary</Palette>
 
 - hex: #121D59
-- rgb: (18, 29, 89)
+- rgb: (18,29,89)
 
-![#A0D1FB](https://placehold.it/15/A0D1FB/000000?text=+) **Secondary**
+<Palette color="#A0D1FB">Secondary</Palette>
 
 - hex: #A0D1FB
 - rgb: (160,209,251)
 
-![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) **Background**
+<Palette color="#FFFFFF">Background</Palette>
 
 - hex: #FFFFFF
-- rgb: (255, 255, 255)
+- rgb: (255,255,255)
 
 #### Typography
 
-![#A0D1FB](https://placehold.it/15/A0D1FB/000000?text=+) **Primary**
+<Palette color="#A0D1FB">Primary</Palette>
 
 - hex: #A0D1FB
-- rgb:
+- rgb: (160,209,251)
 
-![#121D59](https://placehold.it/15/121D59/000000?text=+) **Secondary**
+<Palette color="#121D59">Secondary</Palette>
 
 - hex: #121D59
-- rgb: (18, 29, 89)
+- rgb: (18,29,89)
+
+</TabItem>
 
-“The Free Image Placeholder Service Favoured By Designers.” Placeholder.com, 17 Dec. 2019, placeholder.com/.
+  <TabItem value="dark" label="Dark">
 
 ### Dark Palette
 
-![#A0D1FB](https://placehold.it/15/A0D1FB/000000?text=+) **Primary**
+<Palette color="#A0D1FB">Primary</Palette>
 
 - hex: #A0D1FB
 - rgb: (160,209,251)
 
-![#4f96d8](https://placehold.it/15/4f96d8/000000?text=+) **Secondary**
+<Palette color="#4F96D8">Secondary</Palette>
 
-- hex: #4f96d8
+- hex: #4F96D8
 - rgb: (79,150,216)
 
-![#000000](https://placehold.it/15/000000/000000?text=+) **Background**
+<Palette color="#000000">Background</Palette>
 
 - hex: #000000
 - rgb: (0,0,0)
 
 #### Typography
 
-![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) **Primary**
+<Palette color="#FFFFFF">Primary</Palette>
 
 - hex: #FFFFFF
 - rgb: (255,255,255)
 
-![#121D59](https://placehold.it/15/A0D1FB/000000?text=+) **Secondary**
+<Palette color="#121D59">Secondary</Palette>
 
 - hex: #A0D1FB
 - rgb: (160,209,251)
   <br/>
 
+</TabItem>
+</Tabs>
```

Comment:
Side note: how come Prettier doesn't complain about the formatting? For example, I can see the indents are off

Reply:
I don't know